tests/hook_test.py

```diff
@@ -31,6 +31,25 @@ def tmp_repo(tmpdir):
     subprocess.call(['rm', '-rf', directory])
 
 
+@pytest.fixture
+def tmp_editor(tmpdir):
+    # create EDITOR executable and set environment variable
+    editor_path = os.path.join(str(tmpdir), 'tmp_editor.sh')
+    with open(editor_path, 'w') as f:
+        f.write('#!/bin/bash\n')
+        f.close()
+    subprocess.call(['chmod', '+x', editor_path])
+    old_editor = os.environ.get('EDITOR', None)
+    os.environ["EDITOR"] = editor_path
+
+    yield

```

Comment:
I like this use of vertical whitespace to separate the construction from the destruction.